src/cookieconsent.js

```diff
@@ -34,7 +34,7 @@
       var value = '; ' + document.cookie;
       var parts = value.split('; ' + name + '=');
       return parts.length != 2 ?
-        undefined : parts.pop().split(';').shift();
+        undefined : parts.pop().split(';').shift();crossorigin="anonymous" rel="noreferrer"
```

Comment:
I don't think this works like this. Also please use a new line. Should this be a comment?